packages/main-nav/src/NavContainer.tsx

```diff
@@ -49,11 +49,14 @@ const variantMap = {
 export type NavContainerProps = PropsWithChildren<{
 	id?: string;
 	'aria-label': string;
+	/** Place for your global actions */
```

Comment:
```suggestion
```

Can we remove this comment? Although it's great documentation, it doesn't feel right to document this 1 prop right now and nothing else